### ISSUE TITLE
ImportError: cannot import name 'Iterable' from 'collections' (C:\Python310\lib\collections\__init__.py)

### ISSUE THREAD

**Issue by nyambol** [Calysto/calysto_scheme#36]
Windows 10 Pro

{micha} [122] --> python -V
Python 3.10.0

```
emacs:\org-new\org-mode
{micha} [123] --> pip install --upgrade calysto-scheme --user

emacs:\org-new\org-mode
{micha} [118] --> python -m calysto_scheme install --user
Traceback (most recent call last):
  File "C:\Python310\lib\runpy.py", line 187, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "C:\Python310\lib\runpy.py", line 146, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "C:\Python310\lib\runpy.py", line 110, in _get_module_details
    __import__(pkg_name)
  File "C:\Users\micha\AppData\Roaming\Python\Python310\site-packages\calysto_scheme\__init__.py", line 2, in <module>
    from .kernel import CalystoScheme
  File "C:\Users\micha\AppData\Roaming\Python\Python310\site-packages\calysto_scheme\kernel.py", line 4, in <module>
    from calysto_scheme import scheme
  File "C:\Users\micha\AppData\Roaming\Python\Python310\site-packages\calysto_scheme\scheme.py", line 16, in <module>
    from collections import Iterable
ImportError: cannot import name 'Iterable' from 'collections' (C:\Python310\lib\collections\__init__.py)
```
The kernel is actually installed, but it's broken for the reason shown. On my linux machine, which has Python 3.8, I get this message:

```
Deprecation warning: using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated 
since Python 3.3 and in 3.9 it will stop working.
```

**Comment by nyambol** [comment]
I modified `C:\Users\micha\AppData\Roaming\Python\Python310\site-packages\calysto_scheme\scheme.py`, changing `from collections import Iterable` to `from collections.abc import Iterable` and the issue is fixed. CS is working in a notebook now.  Thanks for providing this great tool.

**Comment by dsblank** [comment]
Thanks for the note! We'll update the code that generates that and create a new version,

**Comment by dsblank** [comment]
Fixed in new version 1.4.7.
